lib/ethon/easy/form.rb

```diff
@@ -21,9 +21,11 @@ class Form
       # @param [ Hash ] params The parameter with which to initialize the form.
       #
       # @return [ Form ] A new Form.
-      def initialize(easy, params)
+      def initialize(easy, params, multipart = nil)
         @easy = easy
         @params = params || {}
+
```

Comment:
Why the newline here?

Reply:
:/... Good question, didn't realize I'd added it. Probably just where I had a comment flagging the changes I'd made in my original verison. I can remove that.